contracts/dex/DEX.sol

```diff
@@ -45,6 +45,46 @@ contract DEX is SystemContract, IDEX {
         return (output[0], output[1]);
     }
 
+    /**
+     * @dev Get Liquidity token address.
+     * Returns (liquidity_token_address)
+     */
+    function getLiquidityTokenAddress(address tokenA, address tokenB)
+    public
+    view
+    override
+    systemContract(tokenA)
```

Comment:
Does it need to be restricted to system contracts? 
Or the erc20 contract created by the user can also be

Reply:
should support any ERC20
but we can only use `currencyId` if it is systemContract